frontend/src/utilities/const.ts

```diff
@@ -46,4 +46,8 @@ export const DEFAULT_CONTEXT_DATA: ContextResourceData<never> = {
 export const REPOSITORY_URL_REGEX =
   /^([\w.\-_]+((?::\d+|)(?=\/[a-z0-9._-]+\/[a-z0-9._-]+))|)(?:\/|)([a-z0-9.\-_]+(?:\/[a-z0-9.\-_]+|))(?::([\w.\-_]{1,127})|)/;
 
-export const DASHBOARD_MAIN_CONTAINER_SELECTOR = 'dashboard-page-main';
+export const DASHBOARD_MAIN_CONTAINER_ID = 'dashboard-page-main';
+
+// Quick starts drawer creates a new scroll container within its DrawerContentBody.
+// Not an ideal selector but components such as JumpLinks require the use of a selector instead of a direct node reference.
+export const DASHBOARD_SCROLL_CONTAINER_SELECTOR = `#${DASHBOARD_MAIN_CONTAINER_ID} > .pf-c-drawer > .pf-c-drawer__main > .pf-c-drawer__content`;
```

Comment:
Here's what we're working with. There's no API to supply a custom ID or class name to the quick starts drawer body content.

The red rectangle represents the current selector.
The green arrow is our scroll container.

![image](https://github.com/opendatahub-io/odh-dashboard/assets/14068621/107960ff-8397-42b5-921d-ed18875d9d6b)